drainer/schema.go

```diff
@@ -78,6 +78,19 @@ func NewSchema(jobs []*model.Job, hasImplicitCol bool) (*Schema, error) {
 	return s, nil
 }
 
+// InitForCreateMySQLSchema create the schema info for `mysql`, since it's created by KV after TiDB 6.2.
+func (s *Schema) InitForCreateMySQLSchema() {
+	db := model.DBInfo{
+		ID:      1,
+		Name:    model.NewCIStr(mysql.SystemDB),
+		Charset: mysql.UTF8MB4Charset,
+		Collate: mysql.UTF8MB4DefaultCollation,
+		State:   model.StatePublic,
+	}
+	s.schemas[1] = &db
```

Comment:
schema id may not be 1 if it is an upgraded cluster

Reply:
How about checking `jobs` and then decide whether to add this job or not?

Reply:
It's ok for now since it's covered by new schema if it is an upgraded cluster